### PR TITLE
RSWEB-7912: Locked content template creation.

### PR DESCRIPTION
**Pull request by amberRrucker**
RSWEB-7912
feat(templates):Locked content template creation

Links to templates are included on the bottom of the page here:
http://localhost:9000/derek/templates/landing-page/

Looking for inclusion of all template portions displayed on the video page here:
https://www.rackspace.com/aw/expertise

and the PDF version here:
https://docs.google.com/drawings/d/1p2Wrgd9cHBb1evPtZfy6cz5QXOvi6cj_DLFEP5tZLtQ/edit

### CODE REVIEW

File: styleguide/derek/templates/landing-page/lockedContent-pdf.ejs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      <div class="col-sm-6">
+        <%- partial("../../_partials/solutions/cta-bottom.ejs") %>
+        <p class="copyright">*Is using cta pattern but can be used with learning button style and small sized title</p>
+      </div>
+    </div>
+  </div>
+
+  <!--Features Pattern-->
+  <div class="bg-light-gray half-padding">
+    <%- partial('../../layouts/features.ejs') %>
```
Comment (WidgetsBurritos) [inline styleguide/derek/templates/landing-page/lockedContent-pdf.ejs]: features.ejs already has a: `<div class="bg-light-gray half-padding">` wrapper. 

image

File: styleguide/derek/templates/landing-page/lockedContent-video.ejs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      <!--Video Pattern Half Width Centered-->
+      <div class="col-sm-6 center-block">
+        <%- partial("../../_partials/solutions/video.ejs") %>
+      </div>
+    </div>
+  </div>
+
+  <!--Features Pattern-->
+  <div class="bg-light-gray half-padding">
+    <%- partial('../../layouts/features.ejs') %>
```
Comment (WidgetsBurritos) [inline styleguide/derek/templates/landing-page/lockedContent-video.ejs]: Same here...

File: styleguide/derek/templates/landing-page/lockedContent-video.ejs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        <div class="col-sm-6">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/cloud/IMAC%20landing%20pages/transforming-it-pdf-cover.jpg" class="center-block">
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- Quadrant -->
+  <div class="container-fluid quadrant" id="quadrant">
+    <div class="container">
```
Comment (WidgetsBurritos) [inline styleguide/derek/templates/landing-page/lockedContent-video.ejs]: We shouldn't have a `container` inside of a `container-fluid`

File: styleguide/derek/templates/landing-page/lockedContent-pdf.ejs [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        <div class="col-sm-6">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/cloud/IMAC%20landing%20pages/transforming-it-pdf-cover.jpg" class="center-block">
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- Quadrant -->
+  <div class="container-fluid quadrant" id="quadrant">
+    <div class="container">
```
Comment (WidgetsBurritos) [inline styleguide/derek/templates/landing-page/lockedContent-pdf.ejs]: We shouldn't have a `container` inside of a `container-fluid`

**Comment by amberRrucker** [comment]
Hey @WidgetsBurritos, Got those changes in for ya.

**Comment by WidgetsBurritos** [comment]
Should these pages be excluded from auto-opening the side nav? 
image

**Comment by amberRrucker** [comment]
@WidgetsBurritos I fixed it! 🔨

**Comment by WidgetsBurritos** [comment]
👍

**Comment by runrog** [comment]
On http://localhost:9000/derek/templates/landing-page/video-LP

Is the video supposed to be butted up to the section below it?

screen shot 2016-12-12 at 10 19 40 am

**Comment by amberRrucker** [comment]
@runrog youre looking at the wrong page. The url is this:
http://localhost:9000/derek/templates/landing-page/lockedContent-video

**Comment by runrog** [comment]
👍  Finally found the super confusing link

**Pull request merged**